rpcserver.go

```diff
@@ -4999,18 +5001,20 @@ func marshalAcceptedQuotes(
 	return rpcQuotes
 }
 
-// QueryRfqAcceptedQuotes queries for accepted quotes from the RFQ system.
-func (r *rpcServer) QueryRfqAcceptedQuotes(_ context.Context,
-	_ *rfqrpc.QueryRfqAcceptedQuotesRequest) (
-	*rfqrpc.QueryRfqAcceptedQuotesResponse, error) {
+// QueryRfqPeerAcceptedQuotes is used to query for quotes that were requested by
+// our node and have been accepted our peers.
+func (r *rpcServer) QueryRfqPeerAcceptedQuotes(_ context.Context,
```

Comment:
nit: since we're renaming: This is located in the `rfqrpc` package, within the `Rfq` service. Do we really need the `Rfq` in this name as well? I think we can shorten some RPC names if we consider that there's always the package name context to help.

Reply:
I'll shorten the name.

I included the RFQ package name because I wanted to make sure that the method would be unique in the `rpcServer` struct method namespace also.

Reply:
That makes sense. Though I think if we ever start to experience naming collisions in the RPC server, then it's time to split that code up into several structs to mirror the packages they implement.